### PR TITLE
fix(Tile): prevent event bubbling and expose event object to callbacks

### PR DESCRIPTION
**Pull request by jzhang300**
**Changelog**

**New**

* Callback events from `Tile` components’ props now have access to `event` object.
* `handleClick` and `handleKeyDown` in the `SelectableTile` component now prevents event bubbling so that they don’t trigger twice.

### CODE REVIEW

**Comment by jzhang300** [comment]
@joshblack can you elaborate what you mean?  Or point me to a line number?

File: src/components/Tile/Tile.js [hunk 1]
```diff
@@ -39,11 +39,11 @@ export class ClickableTile extends Component {
     handleKeyDown: () => {},
   };
 
-  handleClick = () => {
+  handleClick = evt => {
     this.setState({
       clicked: !this.state.clicked,
     });
-    this.props.handleClick();
+    this.props.handleClick(evt);
```
Comment (joshblack) [inline src/components/Tile/Tile.js]: Could we move this to the callback of the `setState` call? e.g.

```js
this.setState(state => ({ clicked: !state.clicked }), () => {
  this.props.handleClick(evt);
});
```

File: src/components/Tile/Tile.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ export class ClickableTile extends Component {
         clicked: !this.state.clicked,
       });
     }
-    this.props.handleKeyDown();
+    this.props.handleKeyDown(evt);
```
Comment (joshblack) [inline src/components/Tile/Tile.js]: Same note as above

**Comment by joshblack** [comment]
Added the notes @jzhang300!

**Comment by jzhang300** [comment]
🎉 I finally get to put my face on that README!

**Pull request merged**